pkg/operation/botanist/component/kubeapiserver/deployment.go

```diff
@@ -724,181 +722,17 @@ func (k *kubeAPIServer) handleVPNSettings(
 	secretCAVPN *corev1.Secret,
 	secretHTTPProxy *corev1.Secret,
 	secretLegacyVPNCAClient *corev1.Secret,
-	secretLegacyVPNSeed *corev1.Secret,
-	secretLegacyVPNSeedTLSAuth *corev1.Secret,
 	secretHAVPNSeedClient *corev1.Secret,
 	secretHAVPNSeedClientSeedTLSAuth *corev1.Secret,
 ) {
-	if !k.values.VPN.ReversedVPNEnabled {
-		k.handleVPNSettingsLegacy(deployment, secretLegacyVPNCAClient, secretLegacyVPNSeed, secretLegacyVPNSeedTLSAuth)
-	} else if k.values.VPN.HighAvailabilityEnabled {
+
+	if k.values.VPN.HighAvailabilityEnabled {
 		k.handleVPNSettingsHAReversedVPN(deployment, secretCAVPN, secretHAVPNSeedClient, secretHAVPNSeedClientSeedTLSAuth)
 	} else {
 		k.handleVPNSettingsReversedVPN(deployment, configMapEgressSelector, secretCAVPN, secretHTTPProxy)
 	}
 }
 
-func (k *kubeAPIServer) handleVPNSettingsLegacy(
-	deployment *appsv1.Deployment,
-	secretLegacyVPNCAClient *corev1.Secret,
-	secretLegacyVPNSeed *corev1.Secret,
-	secretLegacyVPNSeedTLSAuth *corev1.Secret,
-) {
-	deployment.Spec.Template.Labels[v1beta1constants.LabelNetworkPolicyToShootNetworks] = v1beta1constants.LabelNetworkPolicyAllowed
-	deployment.Spec.Template.Spec.InitContainers = []corev1.Container{{
-		Name:  "set-iptable-rules",
-		Image: k.values.Images.AlpineIPTables,
```

Comment:
As far as I see the the `alpine-iptables` image is no longer needed. You could clean up
https://github.com/gardener/gardener/blob/55ffc4bac04a2e068a5d87bbbf4c5cdff287bfba/charts/images.yaml#L201-L204
and the other places related to an image cleanup.

Reply:
Cleaned-up alpine-iptables image references.